frame/vesting/src/lib.rs

```diff
@@ -140,9 +180,15 @@ pub mod pallet {
 		_,
 		Blake2_128Concat,
 		T::AccountId,
-		VestingInfo<BalanceOf<T>, T::BlockNumber>,
+		BoundedVec<VestingInfo<BalanceOf<T>, T::BlockNumber>, T::MaxVestingSchedules>,
 	>;
 
+	/// Storage version of the pallet.
+	///
+	/// New networks start with latest version, as determined by the genesis build.
+	#[pallet::storage]
+	pub(crate) type StorageVersion<T: Config> = StorageValue<_, Releases, ValueQuery>;
+
 	#[pallet::pallet]
 	#[pallet::generate_store(pub(super) trait Store)]
```

Comment:
```suggestion

	#[pallet::pallet]
	#[pallet::generate_store(pub(super) trait Store)]
	#[pallet::generate_storage_info]
```

^ You need to add this, and you might start running into compiler errors. You need to fix those in order to ensure this pallet supports the max encoded length stuff.

Reply:
03bbd59